### PR TITLE
Release 1.5.4

### PR DESCRIPTION
**Pull request by sean-smith**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/stepfunctions/README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# CfnCluster Step Functions
```
Comment (lukeseawalker) [inline cli/stepfunctions/README.md]: wasn't reverted?

Comment (sean-smith) [reply cli/stepfunctions/README.md]: removed.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,13 @@
 CHANGELOG
 =========
 
+1.5.4
+=====
+* Add option to disable ganglia `extra_json = { "cfncluster" : { "ganglia_enabled" : "no" } }``
```
Comment (mauri-melato) [inline CHANGELOG.rst]: extra `

Comment (sean-smith) [reply CHANGELOG.rst]: Fixing in https://github.com/awslabs/cfncluster/pull/525

**Pull request merged**